### ISSUE TITLE
[CoE Starter Kit - QUESTION] Best way to create a non-prod/test environment when you already have one in production

### ISSUE THREAD

**Issue by manuelap-msft** [microsoft/coe-starter-kit#2688]
### What is your question?

What the best way to create a non-prod/test environment when you already have one in production. e.g. copy the environment, create one from scratch, disable before copying, etc.

**Comment by manuelap-msft** [comment]
Additional questions:
- Can Dev and Prod CoE environments run in parallel in a Tenant or are there overlaps (mails are sent twice, etc)?
- Can you explain again which variable I have to set for the "test mode" in the develop environment, so that mails are not sent out to all users? Will such testusecases continue to be expanded?
- Is there an easy way to prevent all emails from being sent to end users. For example, if we are running in a test environment, and want to see the features without impacting users.
- Is it planned can to use variable ProductionEnvironment from Admin Center in all Solutions to test them? If I'm not wrong, not all solutions use this variable?

**Comment by js9331** [comment]
I have a need to create dev/test environments by copying an existing production environment of CoE - is this still an active thread where that information will be shared? Or, should I start a new issue for my question?
